frontend_react/src/api/SocketService.ts

```diff
@@ -0,0 +1,46 @@
+import { SocketServiceInterface } from "../utils/SocketServiceInterface";
+
+const SocketService: SocketServiceInterface = {
+  webSocket: undefined,
+  connect: (websocketAddress: string) => {},
+  disconnect: () => {},
+};
+
+SocketService.connect = (websocketAddress) => {
+  if (SocketService.webSocket === undefined) {
+    SocketService.webSocket = new WebSocket(websocketAddress);
+
+    SocketService.webSocket.onopen = () => {
+      console.log("New client connected!");
+    };
+
+    SocketService.webSocket.onmessage = (event) => {
+      console.log("New message: " + JSON.parse(event.data));
+    };
+
+    SocketService.webSocket.onclose = function (event: CloseEvent) {
```

Comment:
Just bring picky with styling here. It seems like we're using arrow functions throughout the rest of the react repo. Do you also want to convert these to arrow functions for unity sake?